docs/src/main/paradox/index.md

```diff
@@ -15,10 +15,12 @@ connection.
 Based on a protobuf service definition, akka-grpc can generate:
 
 * Model classes (using plain protoc for Java or scalapb for Scala)
-* The API (as an interface for Java or a trait for Scala), expressed in Akka Streams `Source`s
+* The API @scala[trait]@java[interface], expressed in Akka Streams `Source`s
```

Comment:
In the first bullet I figured it would be useful to make the reader aware that we're generating something else for Java than for Scala. 

For the second bullet that didn't seem so important anymore and kind of distracting - but a bit inconsistent, I guess I agree :)